mslib/msui/mpl_qtwidget.py

```diff
@@ -576,31 +577,31 @@ def _determine_ticks_labels(self, typ):
                 labels = ["" if x.split(".")[-1][0] in "9" else x for x in labels]
             ylabel = "pressure (hPa)"
         elif typ == "pressure altitude":
-            bot_km = thermolib.pressure2flightlevel(self.p_bot) * 0.03048
-            top_km = thermolib.pressure2flightlevel(self.p_top) * 0.03048
+            bot_km = thermolib.pressure2flightlevel(self.p_bot * units.Pa).to(units.km).magnitude
+            top_km = thermolib.pressure2flightlevel(self.p_top * units.Pa).to(units.km).magnitude
             ma_dist, mi_dist = 4, 1.0
             if (top_km - bot_km) <= 20:
                 ma_dist, mi_dist = 1, 0.5
             elif (top_km - bot_km) <= 40:
                 ma_dist, mi_dist = 2, 0.5
             major_heights = np.arange(0, top_km + 1, ma_dist)
             minor_heights = np.arange(0, top_km + 1, mi_dist)
-            major_ticks = thermolib.flightlevel2pressure_a(major_heights / 0.03048)
-            minor_ticks = thermolib.flightlevel2pressure_a(minor_heights / 0.03048)
+            major_ticks = thermolib.flightlevel2pressure(major_heights * units.kilometer).magnitude
```

Comment:
kilometer -> km
(consistency and brevity)

Reply:
Done